### PR TITLE
Showcase OpenMP support for Pythran

### PR DESCRIPTION
**Pull request by serge-sans-paille**
Rely on string annotation for explicit parallelism. Use comments instead
of string to avoid disturbing other compilers.

Also increases the pairwise problem size to make the speed difference more
obvious

### CODE REVIEW

File: julia/julia_python.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def julia_python_for_loops(cr, ci, N, bound=1.5, lim=1000., cutoff=1e6):
     '''
     julia = np.empty((N, N), dtype=np.uint32)
     grid_x = np.linspace(-bound, bound, N)
+    #"omp parallel for private(i,x,j,y)"
```
Comment (ogrisel) [inline julia/julia_python.py]: Aren't type comment supposed to follow pep8 as well for consistency? :)

File: pairwise/__init__.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 
-def make_env(shape=(200, 100), seed=0, dtype=np.double):
+def make_env(shape=(300, 150), seed=0, dtype=np.double):
```
Comment (ogrisel) [inline pairwise/__init__.py:7]: The python example will get slower and slower... but I assume this is the only way to measure something interesting.

File: pairwise/pairwise_python.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 def pairwise_python_nested_for_loops(data):
     n_samples, n_features = data.shape
     distances = np.empty((n_samples, n_samples), dtype=data.dtype)
+    #"omp parallel for private(j,d,k,tmp)"
```
Comment (ogrisel) [inline pairwise/pairwise_python.py]: pep8 again

File: pairwise/__init__.py (hunk 2 above)
Comment (serge-sans-paille) [inline pairwise/__init__.py:7]: Yup...

**Comment by serge-sans-paille** [comment]
PEP8 compliance respected!

**Pull request merged**